css/damageTable.css

```diff
@@ -0,0 +1,99 @@
+.damageTable {
+    position: fixed;
+    right: 10px;
+    top: 90px;
+    width: 17em;
+}
+
+.damageTable .control-label {
```

Comment:
.control-label was the bootstrap name, we should probably rename this to something more specific, or if we can't think of anything just "label"